frontend/src/app/main/ui/workspace/context_menu.cljs

```diff
@@ -448,14 +458,11 @@
                     :on-click do-delete}]))
 
 (mf/defc shape-context-menu
+  {::mf/wrap [mf/memo]}
   [{:keys [mdata] :as props}]
   (let [{:keys [disable-booleans? disable-flatten?]} mdata
         shapes (mf/deref refs/selected-objects)
-        hover-ids (mf/deref refs/current-hover-ids)
-        hover-objs (mf/deref (refs/objects-by-id hover-ids))
```

Comment:
It's been puzzling me why this line causes a rerender for a while now, it seems to return the same value on each render, yet it reruns again and again..

Reply:
It caused rerender bucause refs/objects-by-id is always generating a new ref. Is a bit confusing, but we don't need to use hooked version of deref all the times. We don't need always need reactivity on values that we know that will not be changed.